<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <!-- <string>GenderRecognition</string> -->
   <string>FoodClassification</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="imageContainer">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string>No image selected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="buttonContainer">
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>40</number>
       </property>
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="title">
         <property name="font">
          <font>
           <family>Comfortaa</family>
           <pointsize>28</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Food Classification System</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string>QLabel { margin-top: 50px; }</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="selectModel">
         <property name="font">
          <font>
           <family>Abyssinica SIL</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select Model</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="selectImage">
         <property name="font">
          <font>
           <family>Abyssinica SIL</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select Image</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="predict">
         <property name="font">
          <font>
           <family>Abyssinica SIL</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PREDICT</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="outLabel">
         <property name="font">
          <font>
           <family>Abyssinica SIL</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string>background-color: white; border: 1px solid black;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>